xml/s4s_setup_cluster.xml

```diff
@@ -657,11 +663,12 @@
  </sect1>
  <sect1 xml:id="sec-s4s-hana-cluster-wizard-semiautomatic">
   <title>Unattended Setup Using &hana;-SR Wizard</title>
-  
+
   <para>
    To deploy a preconfigured &hana;-SR environments, &suse; offers an
    unattended installation mode for the &hana;-SR wizard.
    To use it, proceed as follows:
+   <remark>sknorr 2019-05-29: mention where I need to perform these steps (what machine).</remark>
```

Comment:
Is need to perform the SAP HANA installation on both nodes.

Reply:
Thanks, @dakechi, but going by the steps:

* Step 1 & 2, running the wizard -- would you do that on some kind of local machine/test machine? Does that machine need a complete HANA installation?

* Step 3 & 4 -- as far as I understand, need to be done on each of the two production machines. Is that correct?

Reply:
@sknorr  1 & 2: Need to be executed on the production machines, as the module tests the existence of IPs, disks, watchdog, etc. And it also needs to have HANA installed on it. The main idea is that the user do it manually on the first time and save the file. If he needs to reproduce it by any reason, like recreating the environment, then he can use the saved configuration to do it automatically.

3 & 4: Is needed to be executed only on the primary node. The module connects to the secondary via SSH and perform the actions there automatically. The yast module RPM package needs to be installed on both nodes previously.